jsm.go

```diff
@@ -682,12 +685,17 @@ func (js *js) AddStream(cfg *StreamConfig, opts ...JSOpt) (*StreamInfo, error) {
 type (
 	// StreamInfoRequest contains additional option to return
 	StreamInfoRequest struct {
+		apiPagedRequest
```

Comment:
I would make this public (I know, that means modifying some other structs that embeds this struct). If we do so, then we may not need a new API (StreamContainedSubjects), but pass &nats.StreamInfoRequest{} as an option to js.StreamInfo(). See test TestStreamInfoSubjectInfo() for an example. The new field "Offset" would be set by the user but then it would be responsibility to the user to call StreamInfo() again with the new offset.

Reply:
Ok so I changed my mind after all about exposing a separate function and instead merged the paging functionality into the existing StreamInfo(): if you pass a value in SubjectFilters in the streamInfoOpts it returns the subject details for the subjects that match (for all the matching subjects and doing the pagination if needed). In the end I think it's the better option/best of both worlds (no new API and as a user I don't need to worry about pagination)